src/Sets/Zonotope/ZonotopeMD/ZonotopeMD.jl

```diff
@@ -0,0 +1,97 @@
+"""
+    struct ZonotopeMD{N, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, DN<:AbstractVector{N}} <: AbstractZonotope{N}
+
+Type that represents a zonotope of order `k` in normal form.
+
+### Fields
+
+- `center::VN` — the center of the zonotope
+- `M::MN` — matrix of general (non-axis-aligned) generators
+- `d::DN` — vector of axis-aligned (diagonal) generators
+
+### Notes
+A zonotope is of order `k` if it has `n * k` generators in `ℝⁿ`, where `n` is the ambient dimension.
+
+A zonotope of order `k` in *normal form* is defined as the set
+
+```math
+Z = \\left\\{ x ∈ ℝ^n : x = c + Mξ + d ⊙ η, ~~ ξ ∈ [-1, 1]^m, ~~ η ∈ [-1, 1]^n \\right\\},
+```
+
+where `M ∈ ℝ^{n×m}` is a matrix of general generators with `m = n*(k -1)` and `d ∈ ℝⁿ` is a vector of axis-aligned generators. 
+Equivalently, this can be seen as a zonotope with generator matrix `[M  D]`, where `D` is the diagonal matrix 
+formed from the vector `d`.
+ZonotopeMD can be constructed in two ways: by passing the full generator matrix `[M  D]` in normal form 
+or by passing `M` and a vector `d` separately.
+
+### Examples
+
+Constructing a zonotope in normal form from a center, general generator matrix `M`, and diagonal vector `d`:
+
+```jldoctest zonotopemd_label
+julia> c = [0.0, 0.0];
+julia> M = [1.0 2.0; 3.0 1.0];
+julia> d = [0.1, 0.2];
```

Comment:
It seems that there must be empty lines after each command. At least that is how it is done [here](https://documenter.juliadocs.org/stable/man/doctests/#REPL-Examples).